style.css

```diff
@@ -0,0 +1,967 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  max-width: 144rem;
+}
+
+body {
+  margin: 0;
+  max-width: 1500px;
+}
```

Comment:
For future projects, please take care of and avoiding these overlapping properties (max-width for body).